xgi/readwrite/xgi_data.py

```diff
@@ -32,15 +39,94 @@ def load_xgi_data(dataset, nodetype=None, edgetype=None, max_order=None):
     XGIError
        The specified dataset does not exist.
     """
-    index_url = "https://raw.githubusercontent.com/ComplexGroupInteractions/xgi-data/main/index.json"
+
+    if read:
+        cfp = _make_unix_file_path(path, dataset)
+        if os.path.exists(cfp):
+            data = json.load(open(cfp, 'r'))
+        else:
```

Comment:
Maybe we could warn the user that the local copy the were looking for does not exist. What do you think?
Maybe a simple print or an actual warning, not sure

Reply:
Sure! Do you want that simply printed or as a warning? Could you suggest a commit for this?

Reply:
```suggestion
        else:
            warn(f"No local copy was found at {cfp}. The data is requested from the xgi-data repository instead. To download a local copy, use `download_xgi_data`.")
```